src/main/kotlin/org/jetbrains/bio/big/Bed.kt

```diff
@@ -179,9 +183,9 @@ data class ExtendedBedEntry(
         val end: Int,
         /** Name of feature. */
         val name: String = ".",
-        /** A number from [0, 1000] that controls shading of item. */
-        val score: Short = 0,
-
+        // UCSC defines score as an integer in range [0,1000], but almost everyone ignores the range.
+        /** Feature score */
+        val score: Int = 0,
```

Comment:
Please update changes notes and mentioned that behavior is fixed

Reply:
CHANGES edited.